Vagrantfile

```diff
@@ -1,86 +1,45 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
-# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
-VAGRANTFILE_API_VERSION = '2'.freeze
-
-Vagrant.require_version '>= 1.5.0'
-
-Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
-  # All Vagrant configuration is done here. The most common configuration
-  # options are documented and commented below. For a complete reference,
-  # please see the online documentation at vagrantup.com.
+def plugin_installed?(name)
+  return if Vagrant.has_plugin?(name)
+  puts "Vagrant plugin '#{name}' is NOT installed."
+  puts 'Please run:'
+  puts "\tvagrant plugin install #{name}"
+  exit(1)
+end
 
-  config.vm.hostname = 'stackstorm-berkshelf'
+# All Vagrant configuration is done below. The '2' in Vagrant.configure
+# configures the configuration version (we support older styles for
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+Vagrant.configure('2') do |config|
+  # For a complete reference, please see the online documentation at
+  # https://docs.vagrantup.com.
 
-  # Set the version of chef to install using the vagrant-omnibus plugin
-  # NOTE: You will need to install the vagrant-omnibus plugin:
-  #
-  #   $ vagrant plugin install vagrant-omnibus
-  #
-  if Vagrant.has_plugin?('vagrant-omnibus')
-    config.omnibus.chef_version = 'latest'
-  end
+  # Every Vagrant development environment requires a box. You can search for
+  # boxes at https://atlas.hashicorp.com/search.
+  config.vm.box = 'bento/centos-7.2'
```

Comment:
It's just most frequently used, most supported and thereby recommended platform by StackStorm team.